src/Bicep.Core.Samples/Files/Variables_LF/main.json

```diff
@@ -218,7 +218,7 @@
     "isFalse": "[not(variables('isTrue'))]",
     "someText": "[if(variables('isTrue'), concat('a', concat('b', 'c')), 'someText')]",
     "scopesWithoutArmRepresentation": {
-      "tenant": "[createObject()]",
+      "tenant": "[tenant()]",
```

Comment:
Worth moving this to another variable (and maybe adding `managementGroup()`) since it's now a valid ARM template function?

Reply:
Good point - done!